ethereum-contracts/README.md

```diff
@@ -1,72 +1,61 @@
+Note: This project currently uses solc@0.5.0, make sure that this version of the Solidity compiler is being used to compile the contracts and does not conflict with other versions that may be installed on your machine.
 
-## Debugging
+## Testing
+Run commands from the appropriate directory: `$ cd ethereum-contracts`  
+Start the truffle environment: `$ truffle develop`  
+In another tab, run tests: `$ truffle test`  
+Run individual tests: `$ truffle test test/<test_name.js>`  
 
-Once you run the contracts you will get the `TX HASH` number of each function call. If not, you can take the `tx` field from the response of the transaction if you print on the respective test file:
+Expected output of the test suite:
+![peggytestsuite](https://user-images.githubusercontent.com/15370712/58388940-34fb6700-7fda-11e9-9aef-6ae7b2442a55.png)
 
-  ```
-  { tx: '0x2e352263105ff582f3c8e5368e3029819a0862f2e3795279c0726b8b8e6c64f7',
-  receipt:
-   { transactionHash: '0x2e352263105ff582f3c8e5368e3029819a0862f2e3795279c0726b8b8e6c64f7',
-     transactionIndex: 0,
-     blockHash: '0x0ae0afbd7d711871e9dde6740ea613bdb22cd9fc4f4c68bac46c22a8bf7e0b5e',
-     blockNumber: 529,
-     gasUsed: 75600,
-     cumulativeGasUsed: 75600,
-     contractAddress: null,
-     logs: [ [Object] ],
-     status: '0x01',
-     logsBloom: '0x00000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000040002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000' },
-  logs:
-   [ { logIndex: 0,
-       transactionIndex: 0,
-       transactionHash: '0x2e352263105ff582f3c8e5368e3029819a0862f2e3795279c0726b8b8e6c64f7',
-       blockHash: '0x0ae0afbd7d711871e9dde6740ea613bdb22cd9fc4f4c68bac46c22a8bf7e0b5e',
-       blockNumber: 529,
-       address: '0x1b5163d8309bc23c225ba666507efd036dc1c677',
-       type: 'someType',
-       event: 'SomeEvent',
-       args: [Object] } ] }
-  ```
+## Security, Privacy, Risks
+Disclaimer: These contracts are for testing purposes only and are NOT intended for production. In order to prevent any loss of user funds, locked Ethereum and ERC20 tokens can be withdrawn directly by the original sender at any time. However, these contracts have not undergone external audits and should not be trusted with mainnet funds. Any use of Peggy is at the user’s own risk.
 
-Copy it and on in a separate tab in the console:
+## Other Considerations
+We decided to temporarily remove the validator set from this version of Peggy, our reasoning being that system transparency and clarity should take precedence over the inclusion of future project features.
```

Comment:
What do you mean? That there still is *a* validator set - just not the same as the Tendermint validator set?

Reply:
Correct. This refers to the validator set from Valset.sol, which was removed from the Smart Contracts for the unidirectional implementation. I've updated the section to be more explicit.